ide-migration/server/migration/api/migration-service.mjs

```diff
@@ -460,6 +460,25 @@ export class MigrationService {
         }
         return projectNames;
     }
+    checkExistingSynonymTypes(locals) {
+        const synonyms = [];
+        for (const local of locals) {
+            const projectSynonymPath = this.getSynonymFilePath(local.projectName);
+            const projectPublicSynonymPath = this.getPublicSynonymFilePath(local.projectName);
+
+            if (local.runLocation === projectSynonymPath) {
```

Comment:
I think these two `if` statements could be combined in `if (local.runLocation === projectSynonymPath || local.runLocation === projectPublicSynonymPath)` and then use `local.runLocation` inside the `if` body